packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -1451,13 +1451,13 @@ exports.questions = {
 	}),
 	planningApplicationDate: () =>
 		new DateQuestion({
-			title: 'What date did you submit your planning application?',
-			question: 'What date did you submit your planning application?',
+			title: 'What date did you submit your application?',
+			question: 'What date did you submit your application?',
 			fieldName: 'onApplicationDate',
 			url: 'planning-application-date',
 			hint: `For example, ${getDate('past')}`,
 			validators: [
-				new DateValidator('the date you submitted your planning application', {
+				new DateValidator('Enter the date you submitted your application', {
```

Comment:
You might want to check the error message here - from memory the DateValidator takes the string but add the 'Enter' at the start so it might not be necessary when invoking the constructor

Reply:
ah ok I'll leave it alone, I thought I was just fixing something obvious but that'll teach me to change copy that I wasn't asked to

Reply:
You might be able to tell that lately I've spent quite a while looking at validators